### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_GettingStartedwithPythoninVSCode_ColorContrast:The color contrast of the numbers in the code section is below 4.5:1.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode-docs#3583]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#CELA P1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.4.3;#Win10;#Visual Studio Code;#CELAP1_code;#ColorContrast;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/python/python-tutorial
Windows Version: Windows10
Browser version: Microsoft Edge Version 81.0.416.60 (Official build) beta (64-bit)

**Repro Steps:**
1)Launch the application url :
code.visualstudio.com/docs/python/python-tutorial 
2)Getting Started with Python in VS Code Page will get open.
3)Go through Install and use packages heading.
4) Launch accessibility insights for desktop and select the third tab "Contrast".

**Actual:**
Navigate to heading "Install and use packages" and "For windows" and hover over the number in green in the code section the color contrast is below 4.5:1.
The ratio is 4.486:1.

**Same issue on the following page:**
1.https://code.visualstudio.com/docs/getstarted/settings
code under heading default settings 
2.https://code.visualstudio.com/docs/python/python-tutorial
under heading verify the Python installation point 2
3.Customization.
4.Editing hacks/IntelliSense.
5.code.visualstudio.com/docs/getstarted/tips-and-tricks site link
6.https://code.visualstudio.com/updates/v1_29#_languages
"outline" text inside the code section under "Improved CSS zero unit warning" heading

**Expected:**
The color contrast of the number should be equal to or greater than 4.5:1.

**User Impact:**
Low vision users will face difficulty in viewly the number with clarity.

**Attachment link for Reference:**
![image](https://user-images.githubusercontent.com/24580570/79853038-27668100-83e5-11ea-8e62-a7714687b26c.png)

**Comment by gregvanl** [comment]
@octref I think I understood that this would be fixed once the website build updated to the latest VS Code Light+ color theme.

**Comment by octref** [comment]
Now seems good to me:

![image](https://user-images.githubusercontent.com/4033249/80523683-4d3ed900-8943-11ea-9191-06e11aebcee2.png)

**Comment by octref** [comment]
Should be fixed in the next published version.

**Comment by Yashi1201** [comment]
Issue is not repro in env https://code.visualstudio.com/docs/python/python-tutorial
Please refer the attachement.
![3583_Issue Fixed](https://user-images.githubusercontent.com/93374043/216590699-9415ada3-aca7-4c7f-b54b-42f1a6ee5a36.png)

Similar Issue:-
1. https://code.visualstudio.com/docs/getstarted/settings
![3583_Similarissue1_Issue Fixed](https://user-images.githubusercontent.com/93374043/216591783-4b4969df-30d0-4fc8-8889-18e48a2584b4.png)

2. https://code.visualstudio.com/docs/python/python-tutorial
![3583_Similarissue2_Issue Fixed](https://user-images.githubusercontent.com/93374043/216593720-47628f89-44e4-4bf7-945a-f8323d17c357.png)

Hence, closing the bug
**GithubTags**:#Closed;
